### DISCUSSION TITLE
Built with FluentUI

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#18332] [Show and tell]
Hi there,

Do we have a page to show-off what we have built using FluentUI?

Thanks.

**Comment by ecraig12345** [comment]
We don't have this at the moment, but it's something that would definitely be cool have in the future (though no concrete plans at the moment). cc @paulgildea

**Comment by blendsdk** [comment]
> We don't have this at the moment, but it's something that would definitely be cool have in the future (though no concrete plans at the moment). cc @paulgildea

I have just created a LinkedIn group for this. Here is the link If anyone is interested: https://www.linkedin.com/groups/9021620/
@paulgildea @ecraig12345

**Reply by JustSlone** [reply]
Thanks for setting this up!
